api/tests/unit/organisations/test_unit_organisation_views.py

```diff
@@ -1497,9 +1497,49 @@ def test_payment_failed_chargebee_webhook_no_subscription_id(
     assert subscription.billing_status == SUBSCRIPTION_BILLING_STATUS_ACTIVE
 
 
+def test_cache_rebuild_event_chargebee_webhook(
+    staff_client: FFAdminUser,
+    mocker: MockerFixture,
+) -> None:
+    # Given
+    data = {
+        "id": "ev_XpbG6hnQqGFm2n3O",
+        "occurred_at": 1341085213,
+        "source": "api",
+        "user": "full_access_key_v1",
+        "object": "event",
+        "api_version": "v2",
+        "content": {
+            "addon": {
+                "id": "support",
+                "name": "Support",
+                "description": "This is addon added when support is needed",
+                "type": "quantity",
+                "charge_type": "recurring",
+                "price": 1000,
+                "period": 1,
+                "period_unit": "month",
+                "status": "deleted",
+            }
+        },
+        "event_type": "addon_deleted",
```

Comment:
Might be overkill (but it would be super trivial to do I think) but should we parametrise this test with all of the cache rebuild event types?

Reply:
Yep, I do tend to agree. Let's leave it as is for now.